modules/contracts/apps/calendar/tests/integration.rs

```diff
@@ -84,74 +83,42 @@ fn request_meeting(
 fn setup_with_time(
     start_time: Time,
     end_time: Time,
-) -> anyhow::Result<(
-    AbstractAccount<Mock>,
-    Abstract<Mock>,
-    AppInterface<Mock>,
-    Mock,
-)> {
-    // Create a sender
-    let sender = Addr::unchecked(ADMIN);
-    // Create the mock
-    let mock = Mock::new(&sender);
-
-    // set balances
-    mock.set_balance(&Addr::unchecked("sender1"), coins(INITIAL_BALANCE, DENOM))?;
-    mock.set_balance(&Addr::unchecked("sender2"), coins(INITIAL_BALANCE, DENOM))?;
-    mock.set_balance(&Addr::unchecked("sender"), coins(INITIAL_BALANCE, DENOM))?;
-
-    // Construct the contract interface
-    let app = AppInterface::new(APP_ID, mock.clone());
-
-    // Deploy Abstract to the mock
-    let abstr_deployment = Abstract::deploy_on(mock.clone(), sender.to_string())?;
-
-    abstr_deployment.ans_host.execute(
-        &abstract_core::ans_host::ExecuteMsg::UpdateAssetAddresses {
-            to_add: vec![(DENOM.to_owned(), AssetInfo::native(DENOM).into())],
-            to_remove: vec![],
-        },
-        None,
-    )?;
-
-    // Create a new account to install the app onto
-    let account =
-        abstr_deployment
-            .account_factory
-            .create_default_account(GovernanceDetails::Monarchy {
-                monarch: ADMIN.to_string(),
-            })?;
-
-    // claim the namespace so app can be deployed
-    abstr_deployment
-        .version_control
-        .claim_namespace(AccountId::local(1), "my-namespace".to_string())?;
-
-    app.deploy(APP_VERSION.parse()?, DeployStrategy::Try)?;
-
-    account.install_app(
-        &app,
+) -> anyhow::Result<(Application<Mock, AppInterface<Mock>>, AbstractClient<Mock>)> {
```

Comment:
We should rename `AppInterface` to `CalendarApp` or `CalendarAppInterface` or something.